intermediate_source/speech_recognition_pipeline_tutorial.py

```diff
@@ -0,0 +1,314 @@
+"""
+Speech Recognition with Torchaudio
+==================================
+
+**Author**: `Moto Hira <moto@fb.com>`__
+
+This tutorial shows how to perform speech recognition using using
```

Comment:
Could you add a colab logo and link to the colab tutorial?

Reply:
Colab link will be generated by the build process of the tutorial. See how [audio tutorial](https://pytorch.org/tutorials/beginner/audio_preprocessing_tutorial.html) has colab link but [the source](https://github.com/pytorch/tutorials/blob/master/beginner_source/audio_preprocessing_tutorial.py) does not.

Meanwhile here is the draft I used. https://colab.research.google.com/drive/1gPv7ugoJsigZrW_4wzhA6TZx7NUm2qQd#scrollTo=Dljo81T6pmKB